src/containers/Board/reducer.js

```diff
@@ -20,7 +20,7 @@ function symbolReducer(board, action) {
   switch (action.type) {
     case ADD_SYMBOL:
       return Object.assign({}, board, {
-        symbols: [...board.symbols, action.symbol]
+        symbols: [...board.symbols, Object.assign({}, action.symbol, { id: board.symbols.length })]
```

Comment:
Hey, good catch, I've been meaning to add unique ids, I will implement with this package: [shortid](https://www.npmjs.com/package/shortid) soon